src/components/common/CustomSelect.js

```diff
@@ -0,0 +1,90 @@
+import React, { PureComponent } from 'react';
+import { string, object, array } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { parseInputErrors } from 'utils/helpers';
+
+export default class CustomSelect extends PureComponent {
+  static propTypes = {
+    input: object.isRequired,
+    label: string,
+    type: string.isRequired,
+    placeholder: string,
+    selectedOption: string,
+    meta: object,
+    options: array
+  };
+
+  constructor() {
+    super();
+    this.state = {
+      options: [],
+      selectedOption: []
+    };
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
Bind not needed. `handleChange` is defined as an arrow function

Reply:
It´s done